app/components/datafruits-player.js

```diff
@@ -110,7 +110,10 @@ export default Component.extend({
         $('.jp-loading').hide();
       },
       solution: 'html, flash',
-      cssSelectorAncestor: ""
+      cssSelectorAncestor: "",
+	  volumechange: function(e){
```

Comment:
I think your editor might be using tabs? I try to keep it consistent across the codebase. No biggie tho